app.py

```diff
@@ -100,178 +101,37 @@ def _project_list_query():
 import app_callbacks
 
 # CREATE APP OBJECT
-load_figure_template(["sandstone", "minty"])
+load_figure_template(["sandstone", "minty", "slate"])
+
+# stylesheet with the .dbc class
+dbc_css = "https://cdn.jsdelivr.net/gh/AnnMarieW/dash-bootstrap-templates/dbc.min.css"
+
+plt_io.templates["custom_dark"] = plt_io.templates["slate"]
+plt_io.templates["custom_dark"]["layout"]["colorway"] = [
+    "#f8dd70",
+    "#c0bc5d",
+    "#8e9b4c",
+    "#62793d",
+    "#3c582d",
+    "#1c381d",
+]
+plt_io.templates.default = "custom_dark"
+
 app = dash.Dash(
     __name__,
     use_pages=True,
-    external_stylesheets=[dbc.themes.SANDSTONE],
+    external_stylesheets=[dbc.themes.SLATE, dbc_css],
```

Comment:
comment why you have these style options in order, if order matters. e.g. "the base color pallet for the app is SLATE and is extended/overridden in some places by our custom dbc_css"

Reply:
tbh on this not sure if the order matters here, the dbc_css a complement to the themes. https://github.com/AnnMarieW/dash-bootstrap-templates see the section on this file